code/modules/power/generators/portable generators/port_gen.dm

```diff
@@ -0,0 +1,86 @@
+
+
+//Baseline portable generator. Has all the default handling. Not intended to be used on it's own (since it generates unlimited power).
+/obj/machinery/power/port_gen
+	name = "Placeholder Generator"	//seriously, don't use this. It can't be anchored without VV magic.
+	desc = "A portable generator for emergency backup power."
```

Comment:
Perhaps a warning should be placed in the description/name/something similar that this is not intended for IC use